main.tf

```diff
@@ -1,3 +1,20 @@
+locals {
+  enabled = module.this.enabled
+  # We do not use coalesce() here because it is OK if local.bucket_name is empty.
+  bucket_name = var.bucket_name == null || var.bucket_name == "" ? module.bucket_name.id : var.bucket_name
```

Comment:
should we use `local.bucket_name` defined here as the s3 module `bucket_name` input?
https://github.com/cloudposse/terraform-aws-lb-s3-bucket/blob/fb07687edd37272bb4cde7755610957e56b15d2b/main.tf#L83